### DISCUSSION TITLE
Ios bugs page doesn't load on ios 15 & ollama stops responding

### DISCUSSION THREAD

**Discussion by AncientMystic** [open-webui/open-webui#2273] [General]
# Bug Report

1. Ios 15 page refuses to load, login loads once logged in the page is blank and i have not found a way to fix this, it loads on ios 17 but for older or not up to date devices it leaves them unable to access open-webui. 

2. All versions of ios if you leave the app (safari or added to home screen bookmark safari app) at all even for a second It immediately stops text generation and often requires ollama to be restarted

## Environment

- Open WebUI Version: 0.1.124 (running in docker) 
- Ollama: 0.1.37 (running directly) 
- Operating System: Ubuntu 20.04.4 LTS 
- docker: 24.0.5 

## Reproduction Details

1. Visit the site on ios 15, login, blank page follows 

2. Begin a text generation with a prompt once assistant begins responding leave the app somehow by switching to a different app leaving the open-webui app running, return to the open-webui window and text generation will have stopped roughly right after you switched windows.

**Comment by snakeying** [comment]
I got the same problem，ios 14.81 page refuses to load, login loads once logged in the page is blank，while 15.81 works fine...wired

Open WebUI Version: v0.1.125

open ai only

Operating System:  centos 7

docker verson : 24.0.7

**Comment by 3377** [comment]
同样问题，作者一直不解决，只能使用librechat

**Comment by 3377** [comment]
https://github.com/open-webui/open-webui/issues/4680
